pkg/odo/cli/service/create.go

```diff
@@ -122,11 +135,25 @@ func (o *ServiceCreateOptions) validateServiceName(i interface{}) (err error) {
 		return err
 	}
 	if exists {
-		return fmt.Errorf("%s service already exists in the current application", o.serviceName)
+		return fmt.Errorf("%s service already exists in the current application", o.ServiceName)
 	}
 	return
 }
 
+// outputNonInteractiveEquivalent outputs the populated options as the equivalent command that would be used in non-interactive mode
+func (o *ServiceCreateOptions) outputNonInteractiveEquivalent() string {
+	if o.outputCLI {
+		var tpl bytes.Buffer
+		t := template.Must(template.New("service-create-cli").Parse(equivalentTemplate))
+		e := t.Execute(&tpl, o)
+		if e != nil {
+			panic(e) // shouldn't happen
```

Comment:
Is there an alternative way to make this more intuitive? Having outputNonInteractiveEquivalent return err? If not, no worries, this still works well, but like your comment, this shouldn't ever happen.